R/writeData.R

```diff
@@ -14,6 +14,7 @@
 #' \code{c(startCol, startRow)}.
 #' @param colNames If \code{TRUE}, column names of x are written.
 #' @param rowNames If \code{TRUE}, data.frame row names of x are written.
+#' @param row.names Deprecated, please use \code{rowNames} instead
```

Comment:
I do not get this. This looks as if you added and deprecated it in the very same moment?

Reply:
`row.names` had been originally passed from `write.xlsx()` and converted to `rowNames` before `writeData()` and `writeDataTable()`.  This was also done with `col.names` and `colNames` but I missed that because the latter wasn't used in testing.

Reply:
I understand. I initially was wondering, why you'd deprecate functions from a stable API, but using similar arguments looks like an improvement. 👍